mimic/rest/maas_api.py

```diff
@@ -333,6 +334,43 @@ def list_entities(self, request, tenant_id):
         request.setResponseCode(200)
         return json.dumps({'metadata': metadata, 'values': entities})
 
+    @app.route('/v1.0/<string:tenant_id>/entities/<string:entity_id>/test-check', methods=['POST'])
+    def test_check(self, request, tenant_id, entity_id):
+        """
+        Replies the test-check
```

Comment:
Can you add a documentation citation for this so that we can easily reference what this API is actually supposed to do?